Mathlib/Probability/Kernel/MeasureCompProd.lean

```diff
@@ -91,19 +96,40 @@ lemma ae_compProd_iff [SFinite μ] [IsSFiniteKernel κ] {p : α × β → Prop}
     (∀ᵐ x ∂(μ ⊗ₘ κ), p x) ↔ ∀ᵐ a ∂μ, ∀ᵐ b ∂(κ a), p (a, b) :=
   Kernel.ae_compProd_iff hp
 
+/-- The composition product of a measure and a constant kernel is the product between the two
+measures. -/
+@[simp]
+lemma compProd_const {ν : Measure β} [SFinite μ] [SFinite ν] :
+    μ ⊗ₘ (Kernel.const α ν) = μ.prod ν := by
+  ext s hs
+  simp_rw [compProd_apply hs, prod_apply hs, Kernel.const_apply]
+
+@[simp]
```

Comment:
I'm not sure this one should be simp (in the same way that `(a + b) * c` is not simped to `a * c + b * c` by default). Same for the next one.